runtime/vm/stringhelpers.cpp

```diff
@@ -445,14 +434,10 @@ verifyQualifiedName(J9VMThread *vmThread, j9object_t string, UDATA allowedBitsFo
 	}
 
 	/* map the return code against the arity value from checkNameImpl (called by bcvCheckClassName):
```

Comment:
This comment seems out of date - the bcv function has no interaction with this code.

Reply:
Removed the outdated comments.